modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/PartitionListener.java

```diff
@@ -70,6 +75,9 @@ public class PartitionListener implements RaftGroupListener {
     /** Logger. */
     private static final IgniteLogger LOG = Loggers.forClass(PartitionListener.class);
 
+    /** Empty sorted set. */
+    private static final SortedSet<RowId> EMPTY_SET = unmodifiableSortedSet(new TreeSet<>());
```

Comment:
It turns out there is `Collections#emptySortedSet()`, sorry for a misleading comment

Reply:
Nice